modules/installation-osp-accessing-api-no-floating.adoc

```diff
@@ -3,10 +3,88 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-kuryr"]
+:osp-kuryr:
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user-kuryr"]
+:osp-kuryr:
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-restricted"]
+:osp-ipi:
+:osp-restricted:
+endif::[]
 
 [id="installation-osp-accessing-api-no-floating_{context}"]
-= Enabling access without floating IP addresses
+= Completing installation without floating IP addresses
+
+// FIP _and_ external network, or FIP _or_ external network?
+You can install {product-title} on {rh-openstack-first} without using a floating IP address and external network. In the
+ifdef::osp-ipi[`install-config.yaml`]
+ifdef::osp-upi[`inventory.yaml`]
+file, leave the following
+ifdef::osp-ipi[parameters]
+ifdef::osp-upi[variables]
+blank:
+
+ifdef::osp-ipi[]
+* `platform.openstack.ingressFloatingIP`
+* `platform.openstack.lbFloatingIP`
+* `platform.openstack.externalNetwork`
+endif::osp-ipi[]
+
+ifdef::osp-upi[]
+* `os_api_fip`
+* `os_ingress_fip`
```

Comment:
`os_bootstrap_fip` variable that needs to be added to this list after https://github.com/openshift/installer/pull/4219.

Reply:
@mandre How does this work for you? https://github.com/openshift/openshift-docs/pull/25216/commits/64d33892eefaf58c4b2f48e3f8515712a074a5a6